### PR TITLE
Improved readme.zh

### PR DESCRIPTION
**Pull request by Steven-WZH**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,5 @@
 # 3X-UI
 
-[English](/README.md) | [中文](/README.zh.md)
```
Comment (MHSanaei) [inline README.md:3]: why you remove this?

File: README.zh.md [hunk 2]
```diff
@@ -1,7 +1,5 @@
 # 3X-UI
 
-[English](/README.md) | [中文](/README.zh.md)
```
Comment (MHSanaei) [inline README.zh.md:3]: and this

**Comment by Steven-WZH** [comment]
I think removing it can make the readme more concise maybe, but that's just my personal opinion

**Comment by Steven-WZH** [comment]
modified

**Pull request merged**